src/chains/ethereum/options/src/wallet-options.ts

```diff
@@ -130,17 +130,27 @@ export type WalletConfig = {
      *
      * @defaultValue false
      */
-    secure: {
+    lock: {
       type: boolean;
       hasDefault: true;
       legacy: {
         /**
-         * @deprecated Use wallet.secure instead
+         * @deprecated Use wallet.lock instead
          */
-        secure: boolean;
+        lock: boolean;
```

Comment:
the legacy name is still `secure`.

```suggestion
        secure: boolean;
```